internal/firefox.py

```diff
@@ -148,6 +149,8 @@ def launch(self, job, task):
             self.marionette.set_window_rect(x=0, y=0, height=task['height'], width=task['width'])
             if 'browserVersion' in self.marionette.session_capabilities:
                 self.browser_version = self.marionette.session_capabilities['browserVersion']
+            if 'moz:buildID' in self.marionette.session_capabilities:
+                self.browser_build = self.marionette.session_capabilities['moz:buildID']
```

Comment:
You can avoid the `if` block using `self.browser_build = self.marionette.session_capabilities.get('moz:buildID')` as `get` will return `None` if the key does not exist. Same applies to browserVersion, above.